templates/list.html

```diff
@@ -6,6 +6,42 @@
      to experience if you were using someone else's built application?
  -->
 
-<div ng-controller="listController">
-  <h1>This is the list page!</h1>
+<div ng-controller="ListController">
+	<div class="container">
+		<h1>IMDB Movie List</h1>
+	</div>
+	<form ng-submit="getMovies()" class="form-inline">
+		<input type="text" class="form-control" id="movieSearchInput" placeholder="Enter Search Here" ng-model="movieText.title" required>
+		<button type="submit" class="btn btn-primary">submit</button>
+	</form>
+	<br>
+	<div class="nav">
+		<div class="container">
+			<ul class="pull-right">
```

Comment:
I loved the font you used for these! One more piece of styling that you could do is that make the mouse a click cursor so its more intuitive for the user to click on those items. Right now when I hover, I dont know if its a clickable button/link or just text. Minor feedback but it works perfectly well!